packages/keepalived/packaging

```diff
@@ -0,0 +1,12 @@
+# abort script on any command that exits with a non zero value
+set -e -x
+
+tar xzvf keepalived/keepalived-1.2.24.tar.gz
```

Comment:
Lets recap our needs:
a- get feedback from community on benefits of this PR without creating too much work for merging or rejecting the PR
b- test the PR on openstack stemcell within orange

For a), lets first open an issue on https://github.com/cloudfoundry-community/cf-haproxy-boshrelease, and reference this PR. Once we're received positive feedback we'll upload the source blobs to the cloudfoundry-community S3 bucket, and PR sent to cloudfoundry-community will reference those blobs, making it easy to merge (similar to https://github.com/cloudfoundry/cf-release/commit/469ffe26e5c9794fc49298278014e930bc9731e6 )

For b), lets temporary upload package sources within the orange S3 elpaaso-bucket within the orange-cf-haproxy-release blob